llama_stack/providers/remote/vector_io/chroma/chroma.py

```diff
@@ -88,7 +88,7 @@ async def delete(self):
 class ChromaVectorIOAdapter(VectorIO, VectorDBsProtocolPrivate):
     def __init__(
         self,
-        config: Union[ChromaVectorIOConfig, ChromaVectorIOConfig],
+        config: ChromaVectorIOConfig,
```

Comment:
The previous code was doing a Union of both the local and remote configs, before the changes in #1023 . Now, this only accepts a Remote config but you'll notice the code below here in `initialize` handles the case of being passed in a remote or local config. This probably still needs to be a Union, and we just need to alias the imports so we can refer to both the local and remote Config types separately here?

Reply:
Line 103 in this file, `log.info(f"Connecting to Chroma server at: {self.config.url}")` - that is where we check for an instance of the remote config, and then it uses the url of that config. Then on line 112, `log.info(f"Connecting to Chroma local db at: {self.config.db_path}")` - that's where we use the path attribute that is only on the local config

Reply:
Fixed!